index.html

```diff
@@ -4,37 +4,42 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Portafolio</title>
 </head>
-
 <body>
     <header>
      <section class="home">
+        <div class="nav-bar">
+            <a href="https://github.com/cvalencia1991">Cesar Valencia</a>
+          <div>
+           <!--menu--> <i class="bi bi-list menu"></i>
+          </div>
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure your **Menu or Hamburger** is wrapped in a `<nav>` tag. You can fix this by just changing the `div` tag to the `nav` tag.

Reply:
ok

Reply:
i got some issues when i put this in my hmtl, that something happend the nav 
image.png